.github/workflows/lint.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: useblacksmith/setup-node@v5
         with:
           node-version: 18.x
-          cache: 'npm'
+          cache: "npm"
```

Comment:
will be inconsistent with gh actions in other repos. can we use single quotes in gh actions? double quotes allow interpolation too so we can use only when necessary?